sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroSource.java

```diff
@@ -294,6 +295,13 @@ public boolean producesSortedKeys(PipelineOptions options) throws Exception {
     return coder;
   }
 
+  @Override
+  public void populateDisplayData(DisplayData.Builder builder) {
```

Comment:
Q: should you be calling `super.populateDisplayData(builder)`? If so, you probably do not need to register anything here.